challenge_03_loops-conditionals/loops-conditionals/while/exercise_00.js

```diff
@@ -0,0 +1,23 @@
+/*
+    1. Create a variable called "loggedIn" that starts off as false.
+    2. Create a while loop that loops while "loggedIn" is false.
+    3. Each loop, print out: "Incorrect login credentials"
+    3. After 3 loop iterations, change "loggedIn" to be true.
+    4. After the loops is done print out: "Sucessfully logged in!"
+
+    HINT1: Be careful for an infinite loop!
+*/
+
+/* RESPONSE */
+
+let loggedIn = false;
+
+while (!loggedIn) {
```

Comment:
This is the output of your script...
```
Incorrect login credentials
Successfully logged in!
```

...have you tested it before pushing? 🤔

⚠️ FIX: needs work.